<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="choose_option_title_label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>611</width>
     <height>85</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>85</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>85</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Pretendard ExtraBold</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color:rgb(153, 204, 255);
</string>
   </property>
   <property name="text">
    <string>Aris 회원 등록</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="name_textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>241</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18px
</string>
   </property>
  </widget>
  <widget class="QLabel" name="phone_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>전화번호</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="phone_textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>241</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18px

</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mb_cancel_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>540</y>
     <width>141</width>
     <height>73</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>73</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Pretendard SemiBold</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
border-radius: 15px;
color:white;</string>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mb_regist_btn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>540</y>
     <width>141</width>
     <height>73</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>73</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Pretendard SemiBold</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(229, 79, 65);
border-radius: 15px;
color:white;</string>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
